src/core/model/table/vertical.h

```diff
@@ -42,7 +42,9 @@ class Vertical {
      * it treats bitsets little endian during comparison and this is not
      * suitable for this case, check out operator< for Columns.
      */
-    bool operator<(Vertical const& rhs) const;
+    bool operator<(Vertical const& rhs) const {
+        return column_indices_ < rhs.column_indices_;
+    }
```

Comment:
Why do we change this operator< behavior?
Given the comment above (`@brief` section), we are not using operator< of bitsets intentionally